x/cronos/types/codec.go

```diff
@@ -20,6 +21,11 @@ func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
 		&TokenMappingChangeProposal{},
 	)
 
+	registry.RegisterImplementations((*sdk.Msg)(nil),
+		&MsgConvertVouchers{},
+		&MsgTransferTokens{},
```

Comment:
```suggestion
		&MsgTransferTokens{},
		&MsgUpdateTokenMapping{},
```

Add `MsgUpdateTokenMapping` together?